cli/util/progress_bar/mod.rs

```diff
@@ -255,17 +277,25 @@ impl ProgressBar {
   }
 
   pub fn update(&self, msg: &str) -> UpdateGuard {
+    self.update_with_prompt(ProgressMessagePrompt::Download, msg)
+  }
```

Comment:
I could have spent some time making our progress bars more generic, but I didn't think it was worth the time atm.

Reply:
As I side note: we should lazily create `ProgressBar` on `ProcState` - it currently shows up in the flamegraphs for each startup when it's checking if we can actually draw. (even if we never use the progress bar, like when only loading local files)

Reply:
I'll lazily create it here

Reply:
Oh wait, not worth it because `IS_TTY_WITH_CONSOLE_SIZE` is already in a lazy and it would have hit that by now.